services-directions/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -532,10 +619,99 @@ public abstract Builder overview(
      * @return this builder for chaining options together
      * @since 3.3.0
      */
-
     @Nullable
     public abstract Builder waypointNames(@Nullable String waypointNames);
 
+    /**
+     * The type of bicycle, either `Road`, `Hybrid`, `City`, `Cross`, `Mountain`. The default type
+     * is Hybrid.
+     *
+     * @param bicycleType the type of bicycle
+     * @return this builder for chaining options together
+     * @since 4.1.0
+     */
+    @Nullable
+    public abstract Builder bicycleType(@Nullable String bicycleType);
```

Comment:
We should add `@DirectionsCriteria.BicycleType` as in https://github.com/mapbox/mapbox-java/blob/a0c9ae45af2fd8b89818aa9660cf447773c944a5/services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java#L842